src/app/components/navigation/SharedToolbar.tsx

```diff
@@ -60,6 +61,15 @@ export const SharedToolbar = (props: SharedToolbarProps): JSX.Element => {
     const sm = useMediaQuery(theme.breakpoints.down('sm'));
     const dispatch = useDispatch();
     const appBarBackground = getScheduledSiteConfig().appBarBackground;
+    const getIsFireworkHoliday = (): boolean => {
+        const holidayClassName = getScheduledSiteConfig().className || '';
+        const fireworkHolidays = ['independence-day'];
+
+        if (fireworkHolidays.includes(holidayClassName)) {
+            return true;
+        }
+        return false;
+    };
```

Comment:
Do we need to have this check and the localstorage item, or are they redundant?

Reply:
I added this check to conditionally render everything before even looking at the local storage and then using the localstorage to turn it on and off. If we absolutely want this on for certain holidays we can probably remove the localstorage stuff across the board. I was mostly putting this in here as an easter egg, assuming it might be a little too much. If we're going to keep it I can refactor (we could use this for new years as well).

Reply:
I say keep it...you can't actually turn it off with localstorage anyway because if we match that holiday you've got code that overwrites the localstorage to on. So if you are in the date range for the theme, you get fireworks no matter what.

Reply:
@joebochill I'm slightly confused by your comment here. I'm assuming the "keep it" means the fireworks in general? Do you want the localstorage stuff in there or do you want me to just set it up so that no matter what we get fireworks if the independence day theme is running?